tracer/src/Datadog.Trace/Debugger/Helpers/TypeExtensions.cs

```diff
@@ -0,0 +1,30 @@
+// <copyright file="TypeExtensions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+namespace Datadog.Trace.Debugger.Helpers
+{
+    internal static class TypeExtensions
+    {
+        internal static bool IsSimple(this Type type)
```

Comment:
I see this is only called in the `SupportedTypesService.cs` file. Maybe it would help resolve the ambiguity here to move it into that file where there are already multiple type checks

Reply:
Agree with you.
I've moved method from extension to the `SupportedTypesService` class at 4b92d59a6fd59efe95c7f637da013d9b919f078b
And we definitely should add some tests on this class.

Reply:
Good point. This method actually should've been declared `private`. I fixed that in a recent commit, then added some tests for the method we do expose (`IsSafeToCallToString`) in99054053ba48b2dc9f9ff1b9bd2650e0a3b52d91